test/dlc_tests/ecs/pytorch/inference/test_ecs_pytorch_inference.py

```diff
@@ -36,6 +36,9 @@ def test_ecs_pytorch_inference_eia(pytorch_inference_eia, ecs_container_instance
     public_ip_address = ec2_utils.get_public_ip(worker_instance_id, region=region)
 
     model_name = "pytorch-densenet"
+    image_framework, image_framework_version = get_framework_and_version_from_tag(pytorch_inference_eia)
+    if image_framework_version is "1.3.1":
```

Comment:
Please use `==` instead of `is` to compare a string variable to a string literal.
```suggestion
    if image_framework_version == "1.3.1":
```

Reasoning:
https://stackoverflow.com/questions/47778889/pylint-complains-about-comparing-a-string-to-a-literal-with-is
https://stackoverflow.com/questions/2209755/python-operation-vs-is-not